src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt

```diff
@@ -108,7 +109,7 @@ class LicenceController(
     @RequestBody @Valid
     request: CreateLicenceRequest,
   ): LicenceCreationResponse = when (request.type) {
-    HARD_STOP -> licenceCreationService.createHardStopLicence(request.nomsId)
+    HARD_STOP, TIME_SERVED -> licenceCreationService.createHardStopLicence(request.nomsId)
     else -> licenceCreationService.createLicence(request.nomsId)
```

Comment:
little odd this as we were bypassing TIME SERVED.. should really be refactored into two seporate methods for HARD_STOP, TIME_SERVED or perhaps just a common named method like create timed out licence.
Obviously the front end was not sending down a type of TIME_SERVED ether. Change in UI PR